src/Microsoft.OData.Edm/Csdl/Parsing/Ast/CsdlAnnotation.cs

```diff
@@ -14,13 +14,15 @@ internal class CsdlAnnotation : CsdlElement
         private readonly CsdlExpressionBase expression;
         private readonly string qualifier;
         private readonly string term;
+        private readonly bool usesDefault; // Note: I may take off this property again if it's not needed for the implementation
 
         public CsdlAnnotation(string term, string qualifier, CsdlExpressionBase expression, CsdlLocation location)
             : base(location)
         {
             this.expression = expression;
             this.qualifier = qualifier;
             this.term = term;
+            this.usesDefault = expression == null; // if the expression is missing, we want the term's default value
         }
 
         public CsdlExpressionBase Expression

```

Comment:
>public CsdlExpressionBase Expression [](http://example.com/codeflow?start=8&length=36)

should add a comment that this will be null if UsesDefault == true. Which means, for this internal class, we may not need the UsesDefault property -- we can just check if Expression == null.  Since it's internal we can update any logic that accesses Expression to understand that null == usesdefault.

Reply:
That makes sense to reduce unnecessary fields! Done ✅